Tests/Tests/CDTURLSessionTests.m

```diff
@@ -55,7 +103,39 @@ - (void)testRequestInterceptorsSetCorrectly
 
     XCTAssertEqual(task.requestInterceptors.count, 1);
     XCTAssertEqual(task.requestInterceptors[0], countingInterceptor);
+    XCTAssertEqual(task.responseInterceptors.count, 1);
+    XCTAssertEqual(task.responseInterceptors[0], countingInterceptor);
     XCTAssertEqual(NSURLSessionTaskStateSuspended, task.state);
+
+    [task resume];
+    while (task.state != NSURLSessionTaskStateCompleted) {
+        [NSThread sleepForTimeInterval:1.0f];
+    }
+    XCTAssertEqual(countingInterceptor.timesCalled, 1);
+    XCTAssertEqual(countingInterceptor.timesResponseIntercepted, 1);
+}
+
+- (void)testInterceptorsCanRerunRequests
```

Comment:
Retry not rerun to match naming elsewhere.

Similarly for "replaying interceptor".